webapp/src/workspace.ts

```diff
@@ -25,14 +25,9 @@ type Header = pxt.workspace.Header;
 type ScriptText = pxt.workspace.ScriptText;
 type WorkspaceProvider = pxt.workspace.WorkspaceProvider;
 type InstallHeader = pxt.workspace.InstallHeader;
```

Comment:
NOTE about this file: this operates on a layer agnostic of the specific workspace; it uses the interfaces